src/controllers/midi/midicontroller.cpp

```diff
@@ -625,14 +625,29 @@ QJSValue MidiController::makeInputHandler(int status, int midino, const QJSValue
         auto mStatusError = QStringLiteral(
                 "Invalid status or midino passed to midi.makeInputHandler. "
                 "Please pass a strictly positive integer. status=%1,midino=%2")
-                                    .arg(status, midino);
+                                    .arg(status)
+                                    .arg(midino);
 
         getScriptEngine()->throwJSError(mStatusError);
         return QJSValue();
     }
 
+    const auto midiKey = MidiKey(status, midino);
+
+    auto it = m_pMapping->getInputMappings().constFind(midiKey.key);
+    if (it != m_pMapping->getInputMappings().constEnd() &&
+            it.value().options.testFlag(MidiOption::Script) &&
+            std::holds_alternative<ConfigKey>(it.value().control)) {
+        qCWarning(m_logBase) << QString(
+                "Ignoring anonymous JS function for status=%1,midino=%2 "
+                "because a previous XML binding exists")
+                                        .arg(status)
+                                        .arg(midino);
```

Comment:
```suggestion
        qCWarning(m_logBase) << QStringLiteral(
                "Ignoring anonymous JS function for status=%1,midino=%2 "
                "because a previous XML binding exists")
                                        .arg(status, midino);
```

Reply:
Erf, no, [that works only for objects that can be casted to QString](https://doc.qt.io/qt-6/qstring.html#arg-14). When used with `(status, midino)`, C++ select [this one](https://doc.qt.io/qt-6/qstring.html#arg-5).

Reply:
I can do:

```suggestion
        qCWarning(m_logBase) << QString(
                "Ignoring anonymous JS function for status=%1,midino=%2 "
                "because a previous XML binding exists")
                                    .arg(QString::number(status), QString::number(midino));
```

Reply:
whoops, good point. yeah, then the double `arg` is probably fine. suboptimal Qt Api. Still, please wrap the string literal in a `QStringLiteral`

Reply:
Done.